src/View/layout.html.twig

```diff
@@ -8,31 +8,37 @@
     <meta http-equiv="x-ua-compatible" content="ie=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="icon" href="/assets/images/favicon.png">
-    <link rel="stylesheet" href="/assets/css/footer.css">
-  
+	<link rel="stylesheet" href="/assets/css/header.css">
+	<link rel="stylesheet" href="/assets/css/footer.css">
+	<link rel="stylesheet" href="/assets/css/contact.css">
+	
+
 
 
 		<title>
 			{% block title %}GooDealMap
 			{% endblock %}
 		</title>
 
+		{% block stylesheet %}{% endblock %}
+
+	</head>
+
 <body>
+   
+
+	{% block header %} 
+	 {% include 'Item/_header.html.twig' %}
+    {% endblock %}
 
-    
-    {% block content %}{% endblock %}
+	{# {% block contact %} 
```

Comment:
Why is there a contact block here ? Contact is a page right?
So it needs a ContactController to render the view